### DISCUSSION TITLE
Is there a mod for disabling the taskbar icons?

### DISCUSSION THREAD

**Discussion by tamr** [ramensoftware/windhawk-mods#963] [Mod ideas]
Subj, for win10 ungrouped taskbar with labels + 7 Taskbar Tweaker.

Some examples:
https://web.archive.org/web/20210713234745/http://des.virtualplastic.net/notaskbarbuttonicon.png
https://web.archive.org/web/20210919115249/http://des.virtualplastic.net/mods.html#sddim

https://web.archive.org/web/20160828063913/https://www.autohotkey.com/board/topic/42514-customize-your-desktop-with-objectivd/
https://www.autohotkey.com/board/topic/78706-tbariconblanker/

**Comment by m417z** [comment]
There's no such mod. Are you willing to hide icons on Windows 10 or 11? For Windows 11, you can try [Windows 11 Taskbar Styler](https://windhawk.net/mods/windows-11-taskbar-styler).
